app/translators/tdr_manifest_to_rawls.py

```diff
@@ -100,12 +101,11 @@ def translate(self, ref_only: bool = False) -> Iterator[Entity]:
                 return self.translate_parquet_file_to_entities(pqfile, False, ref_only)
         elif (parsedurl.scheme == 'https'):
             hostname = parsedurl.netloc
-            if not hostname.endswith(VALID_AZURE_DOMAIN):
+            if not (hostname.endswith(VALID_AZURE_DOMAIN) or hostname == GOOGLE_STORAGE_DOMAIN):
                 logging.error(f"unsupported domain in url {self.filelocation} provided")
                 raise exceptions.InvalidPathException(self.filelocation, user_info, "Unsupported domain")
             with http.http_as_filelike(self.filelocation) as pqfile:
-                # Assumption is that if a file is using https, it's an Azure snapshot
-                return self.translate_parquet_file_to_entities(pqfile, True, ref_only)
+                return self.translate_parquet_file_to_entities(pqfile, hostname.endswith(VALID_AZURE_DOMAIN), ref_only)
```

Comment:
Nit: Using a keyword argument for `is_azure` would make this a little clearer.

```suggestion
                return self.translate_parquet_file_to_entities(pqfile, is_azure=hostname.endswith(VALID_AZURE_DOMAIN), ref_only=ref_only)
```

Reply:
ooh good call.  I'm not super used to python so this is super good to know